src/StatsFuns.jl

```diff
@@ -228,6 +228,9 @@ include("misc.jl")
 include("rmath.jl")
 
 using .RFunctions
+if VERSION >= v"0.6.0-dev.2767"
```

Comment:
Conflict import. You can check CI log. Basically SpecialFunctions isn't precise about importing/extending functions.